api/Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN pip install -r /api/requirements.txt
 COPY . /api
 WORKDIR /api
 
-RUN python -c "from sentence_transformers import SentenceTransformer; SentenceTransformer('all-mpnet-base-v2')"
+ENV TRANSFORMERS_CACHE="/api/big_models_data_cache"
+ENV TORCH_HOME="/api/big_models_data_cache"
```

Comment:
These env vars are THE ONE IMPORTANT CHANGE on this diff, besides the `get_embedder` and `get_highlighter` changes below. Oh and the docker-compose volume shared setup.

You may ignore anything else under `api/scripts`, unless you're curious on the utility scripts we have to populate our database.